addons/pos_self_order/static/src/Components/FloatingButton/FloatingButton.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module */
+
+import { useSelfOrder } from "@pos_self_order/SelfOrderService";
+
+const { Component } = owl;
+
+export class FloatingButton extends Component {
+    static template = "FloatingButton";
```

Comment:
```suggestion
    static template = "pos_self_order.FloatingButton";
```

Make sure of proper namespacing of the template names.